framework/web/JsonResponseFormatter.php

```diff
@@ -38,6 +38,11 @@
 class JsonResponseFormatter extends Component implements ResponseFormatterInterface
 {
     /**
+     * @var string|null the MIME type used in the `Content-Type` header of the response.
+     * When equals `null` default MIME type will be used based on the `useJsonp` property.
+     */
+    public $mimeType = null;
```

Comment:
Maybe we should not automatically add `charset=UTF-8` then and leave it on developer instead?